obal/__init__.py

```diff
@@ -324,7 +324,7 @@ def main(cliargs=None):  # pylint: disable=R0914
     ansible_playbook = (["ansible-playbook"] + ansible_args)
 
     if args.verbose:
-        print(ansible_playbook)
+        print("ANSIBLE_CONFIG={} {}".format(os.environ["ANSIBLE_CONFIG"], ' '.join(ansible_playbook)))
```

Comment:
This is the same, but wondering if it's better.
```suggestion
        print("ANSIBLE_CONFIG={}".format(os.environ["ANSIBLE_CONFIG"]), ' '.join(ansible_playbook))
```

Reply:
Note that I think this is just enough to get you under a 100 characters and make pylint happy.

Reply:
Awesome!